doc/source/advanced.rst

```diff
@@ -206,6 +206,68 @@ Ray supports resource specific accelerator types. The `accelerator_type` field c
 See `ray.util.accelerators` to see available accelerator types. Current automatically detected accelerator types include Nvidia GPUs.
 
 
+Overloaded Functions
+--------------------
+Ray java api support call overloaded java functions remotely. For example, consider the following.
```

Comment:
worth mentioning the reason here. and some minor grammar issues.
```
Ray Java API supports calling overloaded java functions remotely. However, due to the limitation of Java compiler type inference, one must explicitly cast the method reference to the correct function type. ...
```

Reply:
Just found that you've mentioned this below. I think better put it here.

Reply:
fixed